sike-java/src/main/java/com/wultra/security/pqc/sike/crypto/Sike.java

```diff
@@ -68,7 +70,7 @@ public Sike(SikeParam sikeParam, SecureRandom secureRandom) {
      */
     public EncapsulationResult encapsulate(PublicKey pk3) throws GeneralSecurityException {
         if (!(pk3 instanceof SidhPublicKey)) {
-            throw new InvalidKeyException("Invalid public key");
+            throw new InvalidKeyException(INVALID_PUBLIC_KEY);
```

Comment:
Inconsistent code with other exceptions - see bellow. We can rather improve this by creating a single class with constants that we use across the code so that we can reference it like this:

```java
throw new InvalidKeyException(Constants.Exceptions.INVALID_PUBLIC_KEY);
```

Reply:
Added on d4814d815887ec74358c5e52de5770447c1fdcdb.